### PR TITLE
feat: add integration support for common upload.

### PR DESCRIPTION
**Pull request by uzairbaig**

### CODE REVIEW

File: src/main/java/uk/nhs/hee/tis/revalidation/integration/router/service/CommonUploadServiceRouter.java [hunk 1]
```diff
@@ -0,0 +1,39 @@
...
+
+    from("direct:storage-upload")
+        .setHeader(Exchange.HTTP_METHOD, constant(HttpMethod.POST))
+        .setHeader(Exchange.CONTENT_TYPE, constant(MediaType.APPLICATION_JSON))
+        .toD(serviceUrl + API_STORAGE_UPLOAD);
+
+    from("direct:storage-download")
+        .setHeader(Exchange.HTTP_METHOD, constant(HttpMethod.GET))
+        .setHeader(Exchange.CONTENT_TYPE, constant(MediaType.APPLICATION_JSON))
+        .toD(serviceUrl + API_STORAGE_DOWNLOAD);
```
Comment (DorisWongNHS) [inline src/main/java/uk/nhs/hee/tis/revalidation/integration/router/service/CommonUploadServiceRouter.java:31]: May not need to setHeader for GET request

File: src/main/java/uk/nhs/hee/tis/revalidation/integration/router/service/CommonUploadServiceRouter.java [hunk 2]
```diff
@@ -0,0 +1,39 @@
...
+
+    from("direct:storage-list")
+        .setHeader(Exchange.HTTP_METHOD, constant(HttpMethod.GET))
+        .setHeader(Exchange.CONTENT_TYPE, constant(MediaType.APPLICATION_JSON))
```
Comment (DorisWongNHS) [inline src/main/java/uk/nhs/hee/tis/revalidation/integration/router/service/CommonUploadServiceRouter.java:35]: May not need to setHeader for GET request

File: src/main/java/uk/nhs/hee/tis/revalidation/integration/router/service/CommonUploadServiceRouter.java (hunk 1 above)
Comment (DorisWongNHS) [reply src/main/java/uk/nhs/hee/tis/revalidation/integration/router/service/CommonUploadServiceRouter.java:31]: May need to change ."toD" to ".to" if there is requestParam to pass

File: src/main/java/uk/nhs/hee/tis/revalidation/integration/router/service/CommonUploadServiceRouter.java [hunk 3]
```diff
@@ -0,0 +1,39 @@
...
+        .toD(serviceUrl + API_STORAGE_LIST);
```
Comment (DorisWongNHS) [inline src/main/java/uk/nhs/hee/tis/revalidation/integration/router/service/CommonUploadServiceRouter.java:36]: May need to change ."toD" to ".to" if there is requestParam to pass

File: src/main/java/uk/nhs/hee/tis/revalidation/integration/router/service/CommonUploadServiceRouter.java (hunk 2 above)
Comment (uzairbaig) [reply src/main/java/uk/nhs/hee/tis/revalidation/integration/router/service/CommonUploadServiceRouter.java:35]: Done.

File: src/main/java/uk/nhs/hee/tis/revalidation/integration/router/service/CommonUploadServiceRouter.java (hunk 3 above)
Comment (uzairbaig) [reply src/main/java/uk/nhs/hee/tis/revalidation/integration/router/service/CommonUploadServiceRouter.java:36]: Done

File: src/main/java/uk/nhs/hee/tis/revalidation/integration/router/service/CommonUploadServiceRouter.java (hunk 1 above)
Comment (uzairbaig) [reply src/main/java/uk/nhs/hee/tis/revalidation/integration/router/service/CommonUploadServiceRouter.java:31]: Done

**Comment by Judge40** [comment]
@uzairbaig I'm assuming this is redundant now?

**Pull request closed without merging**